pkg/util/hash.go

```diff
@@ -0,0 +1,14 @@
+package k8s
+
+import (
+	"fmt"
+	"hash/fnv"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+)
+
+func Hash(s string) string {
```

Comment:
IMO, based on its usage, it might make more sense for this hash util function to take a namespaced name argument (rather than a string).
Thoughts?